python/pyspark/sql/column.py

```diff
@@ -250,11 +250,39 @@ def __iter__(self):
         raise TypeError("Column is not iterable")
 
     # string methods
+    _rlike_doc = """ Return a Boolean :class:`Column` based on a regex match.\n
```

Comment:
I made a branch without the newlines in `rlike` and `like`. This is the html it makes: https://raw.githubusercontent.com/map222/spark/50fc4f4a4a19a95eb5eaae76840d63f540bd45e0/python/docs/_build/html/pyspark.sql.html
In the html, the `param` line is concatenated to the first one.
Could there be a setting on my local machine that is different?